src/pages/HomePage.tsx

```diff
@@ -24,9 +25,9 @@ const HomePage = (): JSX.Element => {
     <div className="app">
       <header className="header">
         <h1>Kvasir Movies</h1>
-        {user !== null && <p>Welcome back, {user.email}!</p>}
-        {user !== null && <MovieAdder user={user} />}
+        {user && <p>Welcome back, {user.email}!</p>}
```

Comment:
I still prefer `user !== null` over `user`, since then you need to remember what data type user is (any falsey value that has a `toString` will actually render the `toString` version)

Reply:
Not an issue for objects, but as a convention it seems like more overhead to need to rememeber

Reply:
I guess `user != null` feels non-idiomatic to me, and since we have types and we know user is an object, we know we can make that check safely. I think the convention is usually that you shouldn't do that type of check on anything except booleans or objects, so if we keep to that we should be ok? Idk if you feel strongly about it we can switch back

Reply:
I guess I think of it kind of like `==`; it works fine when it's used w/ the right types, but you need to be on-guard if it's used on the wrong types, so it's easier to just say "never use it".

I usually go w/ AirBnB here, but they're kind of strange: https://github.com/airbnb/javascript#comparison--if They say 
> Use shortcuts for booleans, but explicit comparisons for strings and numbers.

Which leaves out objects :/
If the convention is to shortcut for objects as well, I'm fine w/ it